src/lib/pipeline.ml

```diff
@@ -225,16 +228,21 @@ module Full (Bfx: Extended_edsl.Semantics) = struct
   let hla fqs =
```

Comment:
I take it back. It is saved. 

Then minor nit; could you rename the hla -> seq2hla_hla or something?